src/store/Modules/Queries.ts

```diff
@@ -104,9 +105,10 @@ export default class Queries extends VuexModule {
       this.context.commit("updateGlobalQuery", data)
     } catch (err) {
       console.log(err)
+      this.context.commit("updateGlobalQuery", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.